docs/Analyzer Configuration.md

```diff
@@ -452,5 +452,6 @@ Examples:
 
 | Option Value | Summary |
 | --- | --- |
-|`dotnet_code_quality.CA1710.additional_required_suffixes = MyClass->Class` | All types inheriting from MyClass are expected to have the 'Class' suffix |
-|`dotnet_code_quality.CA1710.additional_required_suffixes = MyClass->Class|MyNamespace.IPath->Path` | All types inheriting from 'MyClass' are expected to have the 'Class' suffix AND all types implementing 'MyNamespace.IPath' are expected to have the 'Path' suffix. |
\ No newline at end of file
+|`dotnet_code_quality.CA1710.additional_required_suffixes = MyClass->Class` | All types inheriting from' MyClass' are expected to have the 'Class' suffix |
+|`dotnet_code_quality.CA1710.additional_required_suffixes = MyClass->Class|MyNamespace.IPath->Path` | All types inheriting from 'MyClass' are expected to have the 'Class' suffix AND all types implementing 'MyNamespace.IPath' are expected to have the 'Path' suffix. |
+|`dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader->` | Allows to override built-in suffixes, in this case, all types implementing 'IDataReader' are no longer expected to end in 'Collection' |
```

Comment:
I think we should not require the input to have a trailing `->`, so even the below setting should produce the same result.
```suggestion
|`dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader` | Allows to override built-in suffixes, in this case, all types implementing 'IDataReader' are no longer expected to end in 'Collection' |
```

Reply:
Or maybe for clarity we can allow some special character after the `->`, say `_` or some such?

Reply:
I like the second option.

Reply:
The symbol `_` was a good way to express discard but the problem is that it is a valid class name suffix (in term of what is allowed) even if it's not a nice suffix.